mcc_generated_files/adcc.h

```diff
@@ -71,7 +71,7 @@
 
 typedef uint16_t adc_result_t;
 #ifndef int24_t
-typedef signed short long int int24_t;
+//typedef signed long int int24_t;
```

Comment:
what made it necessary to change the mcc file?

Reply:
issues with compiling

Reply:
and int24_t more or less doesnt seem to be used anywhere that we care about so it wasnt worth trying to save it

Reply:
> issues with compiling
Would be nice to solve the root cause. Like, what's the actual error? Did you google it?